util/LOGGING.md

```diff
@@ -56,6 +56,18 @@ During the request and response processing, any password or authentication token
 util.LogPasswords = true
 ```
 
+It is also possible to skip the output of the some tags (such as the result of `/versions` request,) which are quite large using 
+
+```go
+util.SkipTagList = "SupportedVersions,ovf:License"
+```
+
+For an even more dedicated log, you can define from which function names you want the logs, using
+
+```go
+util.IncludeVersionList = "FindVAppByName,GetAdminOrgByName"
```

Comment:
Why here is called "IncludeVersionList" ?

Reply:
Indeed. And the variable is private. Changed to use the `Set*` functions